src/features/pass-survey/model/AnswerValidator.ts

```diff
@@ -106,6 +106,16 @@ function AnswerValidator(params?: AnswerValidatorArgs): IAnswerValidator {
 
           return true;
         }
+        case 'ParagraphText': {
+          const numberOfCharacter = `${currentAnswer?.answer}`.length;
+          const limit = currentPipelineItem.payload.maxLength;
+
+          if (limit < numberOfCharacter) {
+            return false;
+          }
```

Comment:
Cast `answer` to `ParagraphTextResponse`, fits better with the established pattern in the rest of the validations.

```suggestion
          const answer = currentAnswer?.answer as ParagraphTextResponse;
          const limit = currentPipelineItem.payload.maxLength;

          if (limit < answer.length) {
            return false;
          }
```